ansible_collections/arista/avd/examples/single-dc-multipod-l3ls/README.md

```diff
@@ -0,0 +1,287 @@
+---
```

Comment:
Please add a molecule scenario for this example. Check the other example_xxx scenarios for inspiration. They basically run the example from the example folder, to ensure all vars and artifacts are valid and up to date for every AVD version.
Also remember to add the new molecule scenario to CI workflow and MOLECULE_SCENARIOS file.

Reply:
Done